internal/controller/operatorconfigmap_controller.go

```diff
@@ -950,3 +1002,71 @@ func (c *OperatorConfigMapReconciler) deleteDelegatedCSI() error {
 	}
 	return nil
 }
+
+// TODO: Remove it in 4.20 release
+func (c *OperatorConfigMapReconciler) getCsiCustomTolerations() ([]corev1.Toleration, []corev1.Toleration, error) {
+	rookCephOperatorConfigName, csiPluginTolerationKey, csiProvisionerTolerationKey := "rook-ceph-operator-config", "CSI_PLUGIN_TOLERATIONS", "CSI_PROVISIONER_TOLERATIONS"
+	nodePluginTolerations, controllerPluginTolerations := []corev1.Toleration{}, []corev1.Toleration{}
```

Comment:
why do we need this? you could also just send `nil` for an empty slice and moreover at the caller end do a length check before using, works well when you ack the other comment about clubbing into a single func.